LPS-66783 Using pattern to load soy template

fernandosouza opened the pull request.

marcellustavares committed:
LPS-66476 Modify the LayoutTransformer to provide the fields context instead of rendered fields as part of the pages layout

marcellustavares committed:
LPS-66476 Extract default DDMFormValues creation to a separate class - no logic changes

marcellustavares committed:
LPS-66476 Refactor form template context creation so that the field context carry only information about its state but not its HTML

marcellustavares committed:
LPS-66476 Make DDMFormFieldRenderingContext attributes that are passed to the field template as map

marcellustavares committed:
LPS-66476 Refactor ddm-form-renderer test cases

marcellustavares committed:
LPS-66476 Clean up DDMForm template context creation

marcellustavares committed:
LPS-66476 Clean up DDMFormFieldTypesSerializer to not serialize information no longer needed

sherlyliu committed:
LPS-66857 Birthday cannot be set to date in future.

daledotshan committed:
LPS-66857 SF

Commit:
LPS-66783 Field Settings Sidebar

Commit:
LPS-66783 Renaming files to better understand the context

Commit:
LPS-66783 Removing console.log and syntax fix

Commit:
LPS-66783 Breaking forms CSS rules in smaller files

Commit:
LPS-66783 Overwriting move field behaviour from Alloy Layout Builder

arbetts committed:
LPS-66857 add contact local service test for birthday validation

arbetts committed:
LPS-66857 throw ContactBirthdayException

thiago-rocha commented:
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

sherlyliu committed:
LPS-66857 add contact local service test for parameter is Contact.

Commit:
LPS-66783 Fix show forms has requird fields message

Commit:
Fixing failing tests

Commit:
LPS-66783 The setValue method of the radio field is not a array.

victorg1991 committed:
LPS-67104 New dependency

victorg1991 committed:
LPS-67104 Change implementation

victorg1991 committed:
LPS-67104 Autogen services

Commit:
LPS-67147 - Web content template cacheable option get overriden and unchecked when import template through Resource Importer Web Plugin

victorg1991 committed:
LPS-67104 Change method signature

victorg1991 committed:
LPS-67104 baseLine

drewbrokke committed:
LPS-66857 test - should use updateContact

Commit:
Sidebar merge

marcellustavares committed:
LPS-66476 Fallback to site default locale if there no locale set in the rendering context

marcellustavares committed:
LPS-66476 Update gradle file

marcellustavares committed:
LPS-66476 Set available locales at default form values creation

drewbrokke committed:
LPS-66857 test - use a more drastic date change for the future (all
tests need to fail before changes)

ealonso committed:
LPS-67289 Copy code from email subscriptions settings to subscriptions form navigator entry

ealonso committed:
LPS-67289 Adds form navigator categories for configuration

ealonso committed:
LPS-67289 Adds base form navigator entries for configuration

ealonso committed:
LPS-67289 Copy code from display_settings.jspf to display settings and metadata form navigator entries

Commit:
LPS-66816 Edit the staging group site settings when staging is active

ealonso committed:
LPS-67289 Copy code to selection style form navigator entry

ealonso committed:
LPS-67289 Copy code to ordering and grouping form navigator entry

ealonso committed:
LPS-67289 Show ordering and grouping when we are on dynamic asset selection and portlet is not highest rated assets  or most viewed assets

ealonso committed:
LPS-67289 Show asset entries when we are on manual asset selection

ealonso committed:
LPS-67289 Copy code to asset entries form navigator entry

ealonso committed:
LPS-67289 Copy code to scope form navigator entry

ealonso committed:
LPS-67289 Show scope when we are on manual asset selection or the portlet is not related assets

ealonso committed:
LPS-67289 Copy code to filter form navigator entry

ealonso committed:
LPS-67289 Show filter when we are on dynamic asset selection

Commit:
LPS-67192 - make zero cache value mean SPA cache does not expire

Commit:
LPS-67192 - task-cache

Commit:
LPS-67192 - buildLang

drewbrokke committed:
LPS-67063 Language_fr_CA - uses correctly encoded characters

Commit:
LPS-67192 - use seconds for user input instead of milliseconds to be more intuitive

Commit:
LPS-67192 - buildLang

ealonso committed:
LPS-67201 Rename curTags

ealonso committed:
LPS-67201 Uses new taglib

ealonso committed:
LPS-67201 build.gradle

Commit:
LPS-67184 DDL spreadsheet displays the wrong number of empty rows

ealonso committed:
LPS-67201 Adds matchKey to set the key name

ealonso committed:
LPS-67201 Uses vocabularyIds parameter

ealonso committed:
LPS-67201 No needed

ealonso committed:
LPS-67201 SF

ealonso committed:
LPS-67201 We can have more than one vocabulary

ealonso committed:
LPS-67201 Rename curEntries and curEntryIds

ealonso committed:
LPS-67201 Show only vocabularies with at least one category

adolfopa committed:
LPS-66359 Callers should namespace ID explicitly

Invokers of the tag must ensure that the ID is unique, as the tag won't
namespace it automatically anymore.

adolfopa committed:
LPS-66359 Automatically namespace generated ID

The tag must ensure that any ID it generates is unique. The root problem
is that, despite its name, `PortalUtil.getUniqueElementId()` doesn't
generate unique element IDs; but we cannot fix that without breaking a
lot of stuff, as the broken behaviour is assumed in a lot of places.

Adding the portlet namespace to the generated ID is enough to remove all
known name clashes.

adolfopa committed:
LPS-66359 Don't change provided IDs

When a user of the tags provides an ID, it expects that ID to be
honored, but the tag changes it under the hood. Worse than that,
sometimes is used a namespaced ID, and other times uses the raw one, so
invokers of the tag can only guess what ID should be used in different
cases.

Use the ID as is, and move the responsibility of making these IDs unique
to the tag invoker.

ealonso committed:
LPS-67201 Moves to display context object

nikki-pru committed:
LRQA-26639 Add specific path needed by the test

nikki-pru committed:
LRQA-26639 Fix macro due to switch to alloy editor

nikki-pru committed:
LRQA-26639 Add separate macro to view html content

nikki-pru committed:
LRQA-26639 Rename test to be more representative of what the test is doing

nikki-pru committed:
LRQA-26639 Use new macro

nikki-pru committed:
LRQA-26639 Use correct function

sharonchoi committed:
LPS-66359 Update Wiki macro to use Panel#expand macro for Configuration panel

drewbrokke committed:
LPS-66684 returns type regular role if site is globally scoped

ealonso committed:
LPS-67201 Show only vocabularies with at least one category

ealonso committed:
LPS-67201 Fix source formatter error

ealonso committed:
LPS-67201 Rename file

antonio-ortega committed:
LPS-67225 Fixing class name

drewbrokke committed:
LPS-66857 validate throws ContactBirthday exception

drewbrokke committed:
LPS-66857 updates test

mtambara committed:
LPS-67228 Allow LPKG downgrade

mtambara committed:
LPS-67228 Pull out upgrade logic and make more flexible

mtambara committed:
LPS-67228 Add test for LPKG downgrade

mtambara committed:
LPS-67228 Rename test

Commit:
LPS-67232

antonio-ortega committed:
LPS-67225 Semantic versioning

ealonso committed:
LPS-67289 Removes no needed divs

ealonso committed:
LPS-67201 Avoid duplicated categories

ealonso committed:
LPS-67289 Copy code to source form navigator entry

ealonso committed:
LPS-67289 Copy code to custom form navigator entry

ealonso committed:
LPS-67289 Uses form navigator taglib in configuration

ealonso committed:
LPS-67289 Moves to configuration folder

patriciajeanperez committed:
LPS-60033 Add new locator for the second subject field

patriciajeanperez committed:
LPS-60033 Use unique locator

patriciajeanperez committed:
LPS-60033 Make sure test is calling the correct variables

austinchiang committed:
LRQA-26677 Move KB locators to pathlib files

austinchiang committed:
LRQA-26677 Use pathlib locators for KB macros

austinchiang committed:
LRQA-26677 Deprecate unused KB locators

austinchiang committed:
LRQA-26677 Remove unused KB path file

austinchiang committed:
LRQA-26049 Reorder test steps to prevent autosave from triggering

patriciajeanperez committed:
LRQA-26739 Fix locator

austinchiang committed:
LRQA-26737 Assert text field is present before typing

Commit:
LPS-67184 SF

Commit:
LPS-67192 Default notification timeout should be 30 seconds

sharonchoi committed:
LRQA-26639 Auto SF

vicnate5 committed:
LRQA-26770 Update interactions with xml source

vicnate5 committed:
LRQA-26770 Use Kaleo Designer portlet macros

vicnate5 committed:
LRQA-26771 Update node image

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

ealonso committed:
LPS-67289 Deprecate methods and classes, we should use form navigator

ealonso committed:
LPS-67289 Uses form navigator entry to show the custom user attributes configuration on asset publisher

ealonso committed:
LPS-67289 Removes fieldset because it is no needed

ealonso committed:
LPS-67289 Adds custom form navigator entry to keep the old behaviour of asset entry query processors

ealonso committed:
LPS-67289 Adds new method to check if we have dynamic asset selection or not

ealonso committed:
LPS-67289 Show custom navigator entry only when we are on dynamic asset selection

ealonso committed:
LPS-67289 Adds form navigator id

ealonso committed:
LPS-67289 Adds asset entries form navigator entry

ealonso committed:
LPS-67289 Adds display settings form navigator entry

ealonso committed:
LPS-67289 Adds filter form navigator entry

ealonso committed:
LPS-67289 Adds metadata form navigator entry

ealonso committed:
LPS-67289 Adds scope form navigator entry

ealonso committed:
LPS-67289 Adds source form navigator entry

ealonso committed:
LPS-67289 Adds subscriptions form navigator entry

ealonso committed:
LPS-67289 Adds selection style form navigator entry

ealonso committed:
LPS-67289 Adds ordering and grouping form navigator entry

ealonso committed:
LPS-67289 Adds rss form navigator entry

ealonso committed:
LPS-67289 Copy code from configuration dynamic to rss form navigator entry

ealonso committed:
LPS-67289 Show rss when we are on dynamic asset selection and rss feeds is enabled

Commit:
Revert "LPS-67192 Default notification timeout should be 30 seconds"

This reverts commit be313c235c4c50175b7c12a770ea186dc0f708f0.

Commit:
Revert "LPS-67192 - buildLang"

This reverts commit 86a2a290d9a3d18bb98e508f7d6975ad966e2064.

Commit:
Revert "LPS-67192 - use seconds for user input instead of milliseconds to be more intuitive"

This reverts commit 529124f58eef2303c95368b613fa009863e04d25.

ealonso committed:
LPS-67289 Show custom user attributes only when we are on dynamic asset selection

ealonso committed:
LPS-67289 baseLine

Commit:
LPS-67192 Negative values also means never timing out

Commit:
LPS-67192 task cache

brianchandotcom committed:
LPS-66359 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-settings/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/users-admin/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/portlet-configuration/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site-navigation/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-js/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

hhuijser committed:
LPS-66853 Don't use 'default' version

Commit:
LPS-67147 make all templates cacheable on creation

brianchandotcom committed:
LPS-66853 prep next, use latest

brianchandotcom committed:
LPS-66853 cache

Commit:
LPS-66476 Move servlet to avoid cyclic dependency between dynamic-data-mapping-form-evaluator and dynamic-data-mapping-form-renderer

Commit:
LPS-67300 Increment Marketplace API build number

Commit:
LPS-66476 Clean up DDMFormContextProviderServlet

Commit:
LPS-66476 Fix poshi test

Commit:
LPS-66783 Creating the unic field name based on its label

hhuijser committed:
LPS-66853 Files in service/configuration incorrectly excluded from SF

hhuijser committed:
LPS-66853 Auto SF

Commit:
LPS-66476 Update references to DDMFormContextProviderServlet

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/screens/.gitrepo'.

brianchandotcom committed:
LPS-67300 baseLine

Commit:
LPS-66476 Simplify renderer API by not forcing developer to pass in the context provider servlet url

brianchandotcom committed:
LPS-66853 prep next, use latest

brianchandotcom committed:
LPS-66853 cache

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

Commit:
LPS-66783 Fills down the form fields settings without access the DOM

Commit:
LPS-66476 Use fieldName and instaceId as map key

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

Commit:
LPS-66476 Create cache to ddmFormFieldEvaluationResultsMap property

Commit:
LPS-66476 Add method to get a ddmFormFieldEvaluationResult from a ddmFormEvaluationResult

Commit:
LPS-66783 SF Removing duplicated call of the serializeFromBuilder method

Commit:
LPS-66476 Use the method getDDMFormFieldEvaluationResult instead of getDDMFormFieldEvaluationResultsMap one

Commit:
LPS-66783 SF removing settings modal methods

Commit:
LPS-66783 SF

Commit:
LPS-66476 Delegate fields rendering logic to templates

Commit:
LPS-66476 Apply to form template

Commit:
LPS-66476 Apply to captcha field

Commit:
LPS-66476 Apply to checkbox field

Commit:
LPS-66476 Apply to date field

Commit:
LPS-66476 Apply to editor field

Commit:
LPS-66476 Apply to options field

Commit:
LPS-66476 Apply to paragraph field

Commit:
LPS-66476 Apply to radio field

Commit:
LPS-66476 Apply to select field

Commit:
LPS-66476 Apply to text field

Commit:
LPS-66476 Apply to validation field

Commit:
LPS-66476 Do not try to deserialize inexistent property

Commit:
LPS-66476 Field types need to expose field templateNamespace

Commit:
LPS-66476 Set predefined value for "indexType" and "showLabel" fields

brianchandotcom committed:
LPS-67228 Rename

Commit:
LPS-66476 Render field settings form only when needed through an endpoint rather than always printing it

Commit:
LPS-66476 Fix poshi tests

Commit:
LPS-66476 buildSoy

Commit:
LPS-66476 Frontend tests

brianchandotcom committed:
LPS-66684 Sort

Commit:
LPS-66476 Bump minor version

Commit:
LPS-66476 Update gradle files

brianchandotcom committed:
LPS-66857 SF

brianchandotcom committed:
LPS-66857 SF

brianchandotcom committed:
LPS-66857 Simplify

brianchandotcom committed:
LPS-66857 SF

brianchandotcom committed:
LPS-66857 Rename

brianchandotcom committed:
LPS-66857 auto sort

hhuijser committed:
LPS-66853 Some files get incorrectly excluded from SF

hhuijser committed:
LPS-66853 Auto SF

brianchandotcom committed:
artifact:ignore marketplace-store-web 2.0.4 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-service 2.1.7 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-task-web 1.0.7 prep next

brianchandotcom committed:
artifact:ignore portal-search-web 1.1.5 prep next

brianchandotcom committed:
artifact:ignore export-import-resources-importer 1.1.6 prep next

brianchandotcom committed:
LPS-66853 prep next, use latest

brianchandotcom committed:
LPS-66853 cache

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-mapping/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/portal-workflow/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/marketplace/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

brianchandotcom committed:
LPS-67289 Hugo SF, for now, keep them all at 7.0.0 for new ones. Thx

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

Commit:
LPS-67308 Wrong operator used in String comparison

antonio-ortega committed:
LPS-67225 Comparing the actual portlet to the embedded portlets list.

Commit:
LPS-66934 Move shopping data handler to the proper package

Commit:
LPS-66934 No need to export this

Commit:
LPS-66934 Not needed anymore

jkappler committed:
LPS-67311 Add namespace so that it can be found by js

Commit:
LPS-66783 Sidebar commit merge

Commit:
LPS-66476 Apply to key_value field

brianchandotcom committed:
LPS-67225 SF

Commit:
LPS-66783 Removing old field toolbar class

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

Ithildir committed:
LPS-67314 Use latest gradle-download-task

Ithildir committed:
LPS-67314 cache

brianchandotcom committed:
LPS-67289 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/layout/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

Commit:
LPS-66783 Creates a template variable to the cancel field changes message.

Ithildir committed:
LPS-67278 Do not run "deployAppServerLib" task if not needed

kenjiheigel committed:
LRQA-26767 Increase batch size to 5 to stabilize memory issues on CI for plugins compile test

Commit:
LPS-66476 Fix unit test

rafaprax committed:
LPS-66476 Fix integration tests

Commit:
LPS-66783 Rerendering field settings form when user wants to reset to the previous state.

Commit:
LPS-66783 Updating the value attribute of the validation field when the error message and the parameter change.

Commit:
LPS-66783 Using evaluation started to save the field settings form

Commit:
LPS-66783 SF

Commit:
WIP

Commit:
Renaming sidebar file

Commit:
LPS-66783 Removing undefined function (wrongly solved rebase)

Commit:
LPS-66783 Using tab mode

Commit:
LPS-66783 Indicates if the key value input is editable or not
